src/website/views/components/toolbar/_examples.pug

```diff
@@ -91,7 +91,14 @@ h2 Examples
             <ChiDataTableViews :views="toolbar.viewsData" />
             <div class="chi-divider -vertical"></div>
             <!-- To render filters, use Filters sub-module of Toolbar by providing it with respective data -->
-            <ChiDataTableFilters :filtersData="toolbar.filtersData" />
+            <ChiDataTableFilters :filtersData="toolbar.filtersData" :customItems="toolbar.customItemsData" class="-ml--2">
+              <template v-slot:customAdvanced>
+                <chi-date-picker />
+              </template>
+              <template v-slot:customAdvanced2>
+                <chi-button>Button</chi-button>
```

Comment:
Let's provide another form element or two elements as an example of a filter here